src/SpotifyApi.NetCore.Tests/EpisodeApiTests.cs

```diff
@@ -10,44 +10,37 @@ namespace SpotifyApi.NetCore.Tests
     [TestClass]
     public class EpisodeApiTests
     {
-        [TestCategory("Integration")]
-        [TestMethod]
-        public async Task GetEpisode_EpisodeId_IsNotNull()
+        EpisodesApi api;
+        string bearerAccessToken;
+
+        public EpisodeApiTests()
```

Comment:
Reusing objects in test classes can have unintended consequences. If you want to keep your test code clean, put this code into a method called `Initialize()` with a `[TestInitialize]` attribute. This means the objects get re-initialized on every test, but you can keep your code clean.

Reply:
Hi Daniel,

Thanks did not know this.  Learning a lot as I knew I would from you. Working on all these changes as fast as I can.

Thanks for your patience and time.

Best Regards,

Akshay Srinivasan